pkg/server/apidev/apidev.go

```diff
@@ -364,12 +369,12 @@ func StartViewHandler(ctx context.Context, s *state.State, r *http.Request) (Sta
 	vars := mux.Vars(r)
 	viewSlug, ok := vars["view_slug"]
 	if !ok {
-		return StartViewResponse{}, errors.Errorf("View slug was not supplied, request path must be of the form /dev/startView/<view_slug>")
+		return StartViewResponse{}, libhttp.NewErrBadRequest("view slug was not supplied")
 	}
 
 	viewConfig, ok := s.ViewConfigs.Get(viewSlug)
 	if !ok {
-		return StartViewResponse{}, errors.Errorf("View with slug %s not found", viewSlug)
+		return StartViewResponse{}, libhttp.NewErrBadRequest("View with slug %s not found", viewSlug)
```

Comment:
Prefer %q -- it's better for empty strings 

```suggestion
		return StartViewResponse{}, libhttp.NewErrBadRequest("view with slug %q not found", viewSlug)
```